### PR TITLE
Express-API-Elaine

### PR DESCRIPTION
**Pull request by CoffeeDoodles**

### CODE REVIEW

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
please provide a Heroku link for testing.

**Comment by CoffeeDoodles** [comment]
@KarolCislowski  Ah! Sorry about that I thought I had pushed the last update but... apparently I hadn't, it is now there in the README.

File: README.md [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Learning backend by creating an API using Express that use RESTFUL endpoints.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I created the basic endpoints for returning all the data first and then started to create more specific queries to filter the data. I want to understand how to use .sort so I can alphabetically sort the data rather  than having it appear randomized. I also ran into issues with my ports being used and my ubuntu linux not working properly. But after going through stackoverflow and chatting with a dev mentor I was able to resolve my issues by killing the node process's running and allocating more space and memory to my virtualbox machine.
```
Comment (KarolCislowski) [inline README.md:8]: Short documentation would be nice 🤓

File: server.js [hunk 2]
```diff
@@ -2,29 +2,46 @@ import express from 'express'
...
+    const ramenBrandName = ramenData.filter((ramen) => ramen.brand_name.includes(brand))
+    response.json(ramenBrandName)
+  } if (variety) {
+    const ramenVariety = ramenData.filter((ramen) => ramen.ramen_variety.includes(variety))
+    response.json(ramenVariety)
+  } if (packaging) {
+    const ramenPackaging = ramenData.filter((ramen) => ramen.packaging.includes(packaging))
+    response.json(ramenPackaging)
+  }
+
```
Comment (KarolCislowski) [inline server.js:34]: Nice filtering 👍🏻

**Review by KarolCislowski — APPROVED** [review]
Nice API 👍🏻

**Pull request closed without merging**